Doc/library/argparse.rst

```diff
@@ -1789,7 +1789,7 @@ Sub-commands
    for that particular parser will be printed.  The help message will not
    include parent parser or sibling parser messages.  (A help message for each
    subparser command, however, can be given by supplying the ``help=`` argument
-   to :meth:`add_parser` as above.)
+   to :meth:`!add_parser` as above.)
```

Comment:
```suggestion
   to :meth:`add_parser` as above.)
```

Likewise

Reply:
Or, if we want to implement @AA-Turner 's approach and prepare this for the followup:

```suggestion
   to :meth:`~_SubParsersAction.add_parser` as above.)
```